camlibs/ptp2/library.c

```diff
@@ -6956,6 +6940,8 @@ camera_wait_for_event (Camera *camera, int timeout,
 						free (ob->oi.Filename);
 						C_MEM (ob->oi.Filename = strdup (path->name));
 						strcpy (path->folder,"/");
+						/* TODO: @msmeissn the following goto will leak the `path` memory and
```

Comment:
this came from commit 46adf5b7e4 refactoring. I need to check and rewrite this a bit.